DevOps/DynamoDBAutomation/dynamoDBbuilder.py

```diff
@@ -40,3 +40,15 @@ def create_dynamodb_table(tableName, keyName, cfTemplate):
 
 # return the table status
     return table_status
+
+
+# accept the table name and key name from the user
+tableName = input("Enter the table name: ")
```

Comment:
This is OK for user running the script because the user can enter this information. However, if unattended run is down from a machine, the machine doesn't know how to type on the keyboard and will not be able to provide this information. The goal here is to have the script run like this:

```
python dynamoDBbuilder.py --tableName sometableName --keyName keySchema
```

Reply:
Noted, changes applied.